app/jobs/cache/feed_job.rb

```diff
@@ -1,42 +1,19 @@
 # frozen_string_literal: true
 
 module Cache
-  class FeedJob < ApplicationJob
-    queue_as :cache
-
-    def perform(url, download_media = false)
-      @url = url
-      @download_media = download_media
-
-      @feed = begin
-        benchmark("[Feedjira] #{url}", level: :info) do
-          Feedjira::Feed.fetch_and_parse(@url)
-        end
-      end
-
-      feed_cache_key = Feed.find_by_url(@url).cache_key
-      cached_feed = Rails.cache.fetch(feed_cache_key)
-
-      if cached_feed.blank? || cached_feed.last_modified != @feed.last_modified
-        Rails.cache.write(feed_cache_key, @feed)
-        @updated = true
-      end
-    end
-
+  class FeedJob < Europeana::Feeds::FetchJob
     # Global nav uses some feeds, and is cached so needs to be expired when those
     # feeds are updated.
     # Cached pages need to be expired should they be using the updated feed.
     after_perform do
-      return unless @updated
-      if @download_media
-        @feed.entries.each do |entry|
-          img_url = FeedEntryImage.new(entry).media_object_url
-          DownloadRemoteMediaObjectJob.perform_later(img_url) unless img_url.nil?
+      if @updated
+        if @download_media
```

Comment:
`@download_media` should not be set by `Europeana::Feeds::FetchJob` as it does nothing there.

Scanning this repo's codebase, all uses of `Cache::FeedJob` *do* download media, so this condition can it seems just be removed.

Reply:
good point, these were in here from when exhibitions feeds were dealt with differently to the other feeds. Now that we download media for these two we can remove it.